FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuth.h

```diff
@@ -663,8 +670,10 @@ NS_SWIFT_NAME(Auth)
 - (void)applyActionCode:(NSString *)code completion:(void (^)(NSError *_Nullable error))completion;
 
 /** @fn sendPasswordResetWithEmail:completion:
-    @brief Initiates a password reset for the given email address.
-
+    @brief Initiates a password reset for the given email address. This method does not throw an
```

Comment:
nit: on web SDK, our tech writer suggested - "This method does not throw an error when there's no user account with the given email address and \[Email Enumeration Protection\] is enabled."